jsk_panda_robot/panda_eus/euslisp/dual_panda-interface.l

```diff
@@ -260,7 +260,7 @@
                     ))
   ;; gripper action for real-controller
   (:send-gripper-grasp-action
-   (act width speed force &key (wait t) (inner 0.005) (outer 0.005))
+   (act width speed force &key (wait t) (inner 0.005) (outer 0.06))
```

Comment:
Please also make change in :[start-grasp](https://github.com/softyanija/jsk_robot/blob/2d139ea972114e4062184e98f66a40fcc0d619db/jsk_panda_robot/panda_eus/euslisp/dual_panda-interface.l#L159) and[ :stop-grasp](https://github.com/softyanija/jsk_robot/blob/2d139ea972114e4062184e98f66a40fcc0d619db/jsk_panda_robot/panda_eus/euslisp/dual_panda-interface.l#L166) methods e.g. add arguments in the methods to change inner and outer parameters,